lib/src/base-styles/_nx-colors.scss

```diff
@@ -59,6 +59,39 @@
   --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
   --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-teal-60);
 
+  //NxButton variables
+  --nx-color-primary-button-background: var(--nx-swatch-blue-40);
+  --nx-color-primary-button-background-hover: var(--nx-swatch-blue-30);
+  --nx-color-primary-button-background-active: var(--nx-swatch-blue-50);
+  --nx-color-primary-button-focus-outline: var(--nx-swatch-white);
+  --nx-color-primary-button-text: var(--nx-color-primary-button-focus-outline);
+  --nx-color-secondary-button-background: var(--nx-swatch-white);
+  --nx-color-secondary-button-background-hover: var(--nx-swatch-blue-90);
+  --nx-color-secondary-button-background-active: var(--nx-swatch-blue-95);
+  --nx-color-secondary-button-border: var(--nx-swatch-blue-40);
+  --nx-color-secondary-button-border-active: var(--nx-color-secondary-button-border);
+  --nx-color-secondary-button-focus-outline: var(--nx-swatch-blue-30);
+  --nx-color-secondary-button-text: var(--nx-color-secondary-button-border);
+  --nx-color-secondary-button-text-hover: var(--nx-swatch-blue-30);
+  --nx-color-tertiary-button-background-hover: var(--nx-swatch-indigo-90);
+  --nx-color-tertiary-button-background-active: var(--nx-swatch-indigo-97);
+  --nx-color-tertiary-button-border: var(--nx-swatch-grey-50);
+  --nx-color-tertiary-button-border-active: var(--nx-swatch-grey-60);
+  --nx-color-tertiary-button-focus-outline: var(--nx-swatch-grey-40);
+  --nx-color-tertiary-button-text: var(--nx-color-text-stark);
+  --nx-color-error-button-background: var(--nx-color-alert-error);
+  --nx-color-error-button-background-hover: var(--nx-swatch-red-30);
+  --nx-color-error-button-background-active: var(--nx-swatch-pink-40);
+  --nx-color-error-button-focus-outline: var(--nx-swatch-white);
+  --nx-color-error-button-text: var(--nx-color-error-button-focus-outline);
+  --nx-color-disabled-button-background: var(--nx-color-interactive-background-disabled);
+  --nx-color-disabled-button-border: var(--nx-color-interactive-border-disabled);
+  --nx-color-disabled-button-text: var(--nx-color-text-disabled);
+  --nx-color-primary-segmented-button-divider: var(--nx-swatch-blue-60);
+  --nx-color-button-icon-only-background-active: var(--nx-color-interactive-background-active);
```

Comment:
I'm not seeing this one used anywhere

Reply:
> I'm not seeing this one used anywhere

Thanks for catching that! Removed it here: [28a6d48](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/28a6d48eb4720af4dfbcd9b0464828021c1505ef)